aspnet/fundamentals/localization/sample/Controllers/ManageController.cs

```diff
@@ -0,0 +1,347 @@
+﻿using System;
```

Comment:
I feel the same way about this controller as I do the AccountController, so however it's decided above do the same thing here.